pkgs/games/frogatto/default.nix

```diff
@@ -14,9 +14,10 @@ let
     genericName = "frogatto";
     categories = "Game;ArcadeGame;";
   };
-  version = "unstable-2020-12-04";
+  pname = "frogatto-unstable";
+  version = "2020-12-04";
 in buildEnv {
-  name = "frogatto-${version}";
+  name = "${pname}-${version}"; # i don't know why this is needed
```

Comment:
maybe it's needed because the package is build with `buildEnv`?

without it, the build fails with `name is missing`

Reply:
buildEnv needs a name and does not do pname + version IIRC.